tests/data_context/cloud_data_context/test_checkpoint_crud.py

```diff
@@ -231,7 +231,6 @@ def test_cloud_backed_data_context_get_checkpoint_by_name(
 
 
 @pytest.mark.cloud
-@pytest.mark.unit
```

Comment:
Please leave these existing unit test markers on all of the cloud tests for now.
Otherwise, these won't get run at the desired stage.

We can come back and tweak these again once we have the `cloud` matrix test setup.